### PR TITLE
Fix poster shows '?' when only unaired episodes.

### PR DESCRIPTION
**Pull request by labrys**
Applies same fix as #252 to posters.

### CODE REVIEW

File: gui/slick/views/home.mako [hunk 1]
```diff
@@ -33,7 +33,7 @@
     % if sickbeard.HOME_LAYOUT == 'poster':
     &nbsp;
     <span> Poster Size:
-        <div style="width: 100px; display: inline-block; margin-left: 7px;" id="posterSizeSlider"></div>
```
Comment (VinceVal) [inline gui/slick/views/home.mako:36]: Did you mean to delete this?

**Comment by labrys** [comment]
No and I'm not sure how it got deleted... hmm good catch

**Comment by OmgImAlexis** [comment]
All that was removed was the margin, check the diff.

**Comment by labrys** [comment]
Yeah added it back in #270

**Pull request merged**